app/views/spree/admin/shared/_configuration_menu.html.haml

```diff
@@ -0,0 +1,28 @@
+- content_for :sidebar_title do
+  = Spree.t(:configurations)
+
+- content_for :sidebar do
+  %nav.menu
+    %ul.sidebar
+      = configurations_sidebar_menu_item Spree.t(:general_settings), edit_admin_general_settings_path
+      - if Spree::Config[:override_actionmailer_config]
+        = configurations_sidebar_menu_item Spree.t(:mail_method_settings), edit_admin_mail_method_path
+      = configurations_sidebar_menu_item Spree.t(:image_settings), edit_admin_image_settings_path
+      = configurations_sidebar_menu_item Spree.t(:tax_categories), admin_tax_categories_path
+      = configurations_sidebar_menu_item Spree.t(:tax_rates), admin_tax_rates_path
+      = configurations_sidebar_menu_item Spree.t(:tax_settings), edit_admin_tax_settings_path
+      = configurations_sidebar_menu_item Spree.t(:zones), admin_zones_path
+      = configurations_sidebar_menu_item Spree.t(:countries), admin_countries_path
+      - if Spree::Config[:default_country_id]
+        = configurations_sidebar_menu_item Spree.t(:states), admin_country_states_path(Spree::Config[:default_country_id])
+      = configurations_sidebar_menu_item Spree.t(:payment_methods), admin_payment_methods_path
+      = configurations_sidebar_menu_item Spree.t(:taxonomies), admin_taxonomies_path
+      = configurations_sidebar_menu_item Spree.t(:shipping_methods), admin_shipping_methods_path
+      = configurations_sidebar_menu_item Spree.t(:shipping_categories), admin_shipping_categories_path
+      = configurations_sidebar_menu_item t(:enterprise_fees), main_app.admin_enterprise_fees_path
+      = configurations_sidebar_menu_item Spree.t(:analytics_trackers), admin_trackers_path
+      = configurations_sidebar_menu_item t('admin.cache_settings.show.title'), main_app.admin_cache_settings_path
+      = configurations_sidebar_menu_item t('admin.contents.edit.title'), main_app.edit_admin_content_path
+      = configurations_sidebar_menu_item t('admin.invoice_settings.edit.title'), main_app.edit_admin_invoice_settings_path
+      = configurations_sidebar_menu_item t('admin.matomo_settings.edit.title'), main_app.edit_admin_matomo_settings_path
+      = configurations_sidebar_menu_item t('admin.stripe_connect_settings.edit.title'), main_app.edit_admin_stripe_connect_settings_path
```

Comment:
I didnt want to touch translations in this PR. We should probably remove all Spree.t entries (there are more) from ofn's code and move everything we can to our namespaces.